python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -397,6 +397,13 @@ def main():
         sys.stderr.write('"multiprocessing" missing.\n')
         exit_code = 16
 
+    try:
+        import scandir
+        scandir.__file__
```

Comment:
Ok, so what do we need to check? Both `scandir` and `_scandir`? Or only `_scandir`?

Here's how things look on a build slave where `scandir` is compiled through PIP_LIBRARIES.

```
[buildslave@bs1a-fbsd-10stable-x64-44 ~/runtime/build-freebsd10-x64/slave/python-package-fbsd-10/build/build]$ ./python2.7-freebsd10-x64/bin/python
Python 2.7.10 (caf31f3, Feb 27 2017, 21:27:16)
[GCC 4.2.1 Compatible FreeBSD Clang 3.4.1 (tags/RELEASE_34/dot1-final 208032)] on freebsd10
Type "help", "copyright", "credits" or "license" for more information.
>>> import scandir
>>> scandir.__file__
'/srv/buildslave/runtime/build-freebsd10-x64/slave/python-package-fbsd-10/build/build/python2.7-freebsd10-x64/lib/python2.7/site-packages/scandir.pyc'
>>> import _scandir
>>> _scandir.__file__
'/srv/buildslave/runtime/build-freebsd10-x64/slave/python-package-fbsd-10/build/build/python2.7-freebsd10-x64/lib/python2.7/site-packages/_scandir.so'
```

And here's how things look on a build slave where `scandir` is compiled through EXTRA_LIBRARIES_NO_CFFI:
```
buildslave@bs1i-lnx-sles10sp3-x64-47:~/runtime/build-sles10-x64/slave/python-package-sles-10/build/build> ./python2.7-sles10-x64/bin/python
Python 2.7.10 (caf31f3, Feb 27 2017, 20:43:24)
[GCC 4.1.2 20070115 (SUSE Linux)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
Traceback (most recent call last):
  File "/etc/pythonstart", line 7, in <module>
    import readline
ImportError: No module named readline
>>> import scandir
>>> scandir.__file__
'/srv/buildslave/runtime/build-sles10-x64/slave/python-package-sles-10/build/build/python2.7-sles10-x64/lib/python2.7/site-packages/scandir-1.5-py2.7-linux-x86_64.egg/scandir.pyc'
>>> import _scandir
>>> _scandir.__file__
'/srv/buildslave/.python-eggs/scandir-1.5-py2.7-linux-x86_64.egg-tmp/_scandir.so'
```